.travis.yml

```diff
@@ -81,10 +85,10 @@ before_script:
   - (cd examples/truffle; yarn --frozen-lockfile && yarn build)
 
 script:
-  - cargo build --workspace --all-features --verbose
-  - cargo test --workspace --all-features --verbose
   - cargo fmt -- --check
   - cargo clippy --workspace --all-features --all-targets -- -D warnings
+  - cargo build --workspace --all-features --verbose
+  - cargo test --workspace --all-features --verbose
```

Comment:
Any special reason the order was switched? My guess is that this ordering is more optimal if travis short-circuits on a `fmt` or `clippy` failure

Reply:
:+1: